ingestion/src/metadata/utils/source_connections.py

```diff
@@ -111,7 +111,9 @@ def get_connection_url_common(connection):
         else connection.connectionOptions
     )
     if options:
-        if not connection.database:
+        if (hasattr(connection, "database") and not connection.database) or (
```

Comment:
thinking out loud here, would this be the same as `not getattr(connection, "database")`?

Reply:
umm, I guess `hasattr(connection, "database") and connection.database` this will be similar to `getattr(connection, "database") ` so `not  getattr(connection, "database")` would make it `not hasattr(connection, "database") and  not connection.database` 🤔

Reply:
not the best mental state to think about booleans, you're right. Did some manual tests and cannot figure out a one-liner hehe, we can keep it like this

for reference:
<img width="770" alt="image" src="https://user-images.githubusercontent.com/35870520/178025847-7cd3de74-6c99-4cd3-9b89-98bd86361bbf.png">